syntax="proto3"; //消息类型命名规范：使⽤驼峰命名法，⾸字⺟⼤写。
package myprotobuf;

message MyMessage{
    oneof test_oneof { 
        int32  mynumber1 = 1;
        int32  mynumber2 = 2;
    }
}

